descabi-morello/descabi-morello.rst

```diff
@@ -0,0 +1,769 @@
+    +------------+----------+----------------------------------------------------------------------------------------------------+
+    | r28        |          | The Capability Private Data Addressing Register.                                                   |
+    +------------+----------+----------------------------------------------------------------------------------------------------+
+    | r21-r27    |          | Registers r21-r27 (c21-c27) are callee-saved.                                                      |
+    +------------+----------+----------------------------------------------------------------------------------------------------+
+    | r19-r20    |          | Registers r19-r20 (c19-c20) are callee-saved for the callee and temporaries for the caller.        |
+    +------------+----------+----------------------------------------------------------------------------------------------------+
+    | r18        |          | The Platform Register, if needed; otherwise a temporary register.                                  |
+    +------------+----------+----------------------------------------------------------------------------------------------------+
+    | r17        |          | The Capability Frame Register.                                                                     |
+    +------------+----------+----------------------------------------------------------------------------------------------------+
+    | r16        | CIP0     | The first intra-procedure-call scratch register (can be used by call veneers and PLT code).        |
+    +------------+----------+----------------------------------------------------------------------------------------------------+
+    | r9-r15     |          | Temporary registers.                                                                               |
+    +------------+----------+----------------------------------------------------------------------------------------------------+
+    | r8         |          | The capability indirect result location register.                                                  |
+    +------------+----------+----------------------------------------------------------------------------------------------------+
+    | r0-r7      |          | Parameter/result registers.                                                                        |
+    +------------+----------+----------------------------------------------------------------------------------------------------+
+
+
+Global entry points
+-------------------
+
+Each function has an additional entry point (the global entry point) that is used to manage
+the value of the Capability Private Data Addressing Register. The global entry point is a thunk
+generated by the linker and is used for indirect calls or calls from a different DSO.
+The global entry point is located in the same DSO as the associated function.
```

Comment:
What section?

Reply:
Not sure we need a specific section here, as long as it these end up in the pcc bounds.